packages/styles/components/_c.option-card.scss

```diff
@@ -0,0 +1,94 @@
+@use "sass:math";
+
+.mc-option-card {
+  $parent: get-parent-selector(&);
+
+  position: relative;
+  border-radius: get-border-radius("m");
+  cursor: pointer;
+
+  &__input {
+    position: absolute !important;
```

Comment:
The `!important` is not necessary.

~~border-color: $color-input-border;~~ <= not necessary